sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## 2.0.0-beta.4 (Unreleased)
 
-## Bug fixes
+### New features
+
+- Added `loginState`, `loginNonce`, `loginDomainHint`, `loginExtraQueryParameters`, `loginRedirectStartPage`, `loginOnRedirectNavigate`, `loginClaims` to the constructor options of the browser version of the `InteractiveBrowserCredential`, to mitigate cross site request forgery attacks and multiple replay attacks.
```

Comment:
It's not clear to me how I would use these new options to mitigate security risks

Reply:
I’ll try look for samples!

Reply:
We’re asking across teams now, since the available documentation seems to not have clear samples we can copy. I’ll come back probably next week with more information!